src/QsCompiler/Compiler/CompilationLoader.cs

```diff
@@ -127,8 +142,8 @@ private class ExecutionStatus
                 WasSuccessful(options.SerializeSyntaxTree, this.Serialization) &&
                 WasSuccessful(options.BuildOutputFolder != null, this.BinaryFormat) &&
                 WasSuccessful(options.DllOutputPath != null, this.DllGeneration) &&
-                !this.BuildTargets.Values.Any(status => !WasSuccessful(true, status))
-                ? 0 : 1;
+                !this.LoadedRewriteSteps.Values.Any(status => !WasSuccessful(true, status))
+                ? true : false;
```

Comment:
Shouldn't the result already be bool? If so, this this line might not be necessary.

Reply:
Oh boy, yes you are right. :)